apps/solution/src/components/people/people.tsx

```diff
@@ -37,7 +37,7 @@ export interface SearchBarProps {
 function SearchBar({ searchTerm, onSearchTermChange }: SearchBarProps) {
 
   return (
-    <div className={style.searchBar}>
+    <div className={style.searchBar} data-cy="searchbar">
```

Comment:
I think browser actually have dedicated `search` aria-label, especially useful for accessibility, and I think it's a good practice to use them for testing purpose too.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/search_role

Could you please give it a try and let me know? I'm very interested of this! Thanks

Reply:
I changed the div to a form, put a role search and an aria label describing what it searches. 
Seem to be the best practice here.
I therefore needed to preventDefault onSubmit

Select seem to work with the role.